src/Framework/TestSuite.php

```diff
@@ -292,6 +292,12 @@ public function addTestSuite($testClass)
      */
     public function addTestFile($filename)
     {
+        static $declaredClasses;
```

Comment:
Please do not use a method-local, static variable here.

Reply:
@sebastianbergmann done, changed to private, dynamic one.